src/JBController.sol

```diff
@@ -60,6 +60,7 @@ contract JBController is JBPermissioned, ERC2771Context, IJBController, IJBMigra
     error JBController_InvalidReservedPercent(uint256 percent, uint256 limit);
     error JBController_MintNotAllowedAndNotTerminalOrHook();
     error JBController_NoReservedTokens();
+    error JBController_OnlyFromTargetTerminal(address sender, IERC165 targetTerminal);
```

Comment:
This `IERC165` type is a bit strange in this situation imo, we could have this be an `IJBProjectUriRegistry` but just having it be `address`  would make more sense to me, since the address potentially(/likely) is not an `IJBProjectUriRegistry` if this error occurs.